docs/guides/unreachable.rst

```diff
@@ -54,7 +54,7 @@ As an example, consider this simple calculator:
 .. code:: python
 
    import enum
-   from typing_extensions import Never
+   from typing import Never
```

Comment:
Never needs an infobox as well. (Added in Python 3.11.)

Reply:
I think I didn't add it for that one because it's already explained in the prose at the start of the page; but I've added an additional infobox below the code snippet now for consistency (and perhaps easier discoverability for people who only skim the page) :+1: